Packages/vcs/Lib/utils.py

```diff
@@ -1716,6 +1716,37 @@ def getworldcoordinates(gm, X, Y):
     return wc
 
 
+def rgba_color(color, colormap):
+    """Try all of the various syntaxes of colors and return 0-100 RGBA values."""
+    try:
+        # Is it a colormap index?
+        return colormap.index[color]
+    except ValueError:
```

Comment:
Interesting, I would expect a IndexError from this code but your code looks go to me.